meeting-minutes/2025-03-11.md

```diff
@@ -0,0 +1,126 @@
+# Attendees
+
+- Jessica Tallon (**Editor**)
+- Christine Lemmer-Webber
+- Dan C (**Scribe**)
+- David Thompson
+- Jonathan Rees (**Chair**)
+- Kris Kowal (**Editor**)
+- Mark Miller
+- Richard Gibson
+
```

Comment:
Usually we format it with people's affiliation in parentheses and the chair and scribe broken out, like this: https://github.com/ocapn/ocapn/blob/main/meeting-minutes/2024-11-12.md